lib/govuk_tech_docs/tech_docs_html_renderer.rb

```diff
@@ -69,7 +69,7 @@ def table_row(body)
         first_child.content = leading_text.sub(/# */, "")
       end
 
-      tr = Nokogiri::XML::Node.new "tr", fragment
+      tr = Nokogiri::XML::Node.new "tr", fragment.document
```

Comment:
this change can be tested by viewing:
http://localhost:4567/code.html

If you checkout just the first commit you will see the deprecation warning, if you then apply the second commit you will see it fixed